vars/peon_defaults.yml

```diff
@@ -54,23 +54,6 @@ peon_defaults:
 
     fedora:
       volume: "20"
-      version: "24"
-      groups:
-        - fedora
-        - autotested
-        - peons
-        - updated
-
-    fedoratomic:
-      version: "24"
-      groups:
-        - fedoratomic
-        - atomic_upgrade
-        - autotested
-        - peons
-
-    fedora:
-      volume: "20"
```

Comment:
Technically we could do both 24 and 25 - it just requires changing the names (``fedoratomic -> fedoratomic24 and fedoratomic25``).  But why bother, easier all around to only test one Fedora.